src/components/Filters/ViewsMenu.tsx

```diff
@@ -176,6 +181,7 @@ export interface ViewsMenuProps {
 	setFilters: (filters: JSONSchema6[]) => void;
 	deleteView: (view: FiltersView) => any;
 	renderMode?: FilterRenderMode | FilterRenderMode[];
+	shouldHideLabel: boolean;
```

Comment:
Normally we use positive names, such as `shouldShowLabel`, but it's fine like this as I guess it matches the values from the `compact` array better.

Reply:
absolutely agree, usually I use positive names too. In this case I used it this way to match the `compact` prop in a better way